topics/lua-api.md

```diff
@@ -377,6 +377,18 @@ This function triggers a breakpoint when using the Redis Lua debugger](/topics/l
 
 This function prints its argument in the [Redis Lua debugger](/topics/ldb) console.
 
+### <a name="redis.acl_check_cmd"></a> `redis.acl_check_cmd(command [,arg...])`
+
+* Since version: 7.0.0
+* Available in scripts: yes
+* Available in functions: yes
+
+This function is used to check if the current user running the script has [ACL](/topics/acl) permissions to execute the given command with the given arguments.
+
+The return value is a boolean `true` in case the current user has permissions to execute the command (via a call to [reids.call](#redis.call) or [redis.pcall](#redis.pcall)) or `false` in case they don't.
+
+The function will raise an error if the passes command or its arguments are not a valid Redis command.
```

Comment:
```suggestion
This function is used for checking if the current user running the script has [ACL](/topics/acl) permissions to execute the given command with the given arguments.

The return value is a boolean `true` in case the current user has permissions to execute the command (via a call to [redis.call](#redis.call) or [redis.pcall](#redis.pcall)) or `false` in case they don't.

The function will raise an error if the passed command or its arguments are invalid.
```